src/docfx/config/ContributionConfig.cs

```diff
@@ -37,5 +39,10 @@ internal class ContributionConfig
         /// It should be an absolute url or a relative path
         /// </summary>
         public readonly string GitCommitsTime = string.Empty;
+
+        /// <summary>
+        /// The excluded contributors which you do want to show
+        /// </summary>
+        public readonly string[] ExcludedContributors = Array.Empty<string>();
```

Comment:
This can be a `Hashset<string>` so you don't need an additional hashset in ContributionInfo

Reply:
Then the user may got errors, i perfer to preprocess it

---
In reply to: [210815881](https://github.com/dotnet/docfx/pull/3224#discussion_r210815881) [](ancestors = 210815881)